config/dev-local.js

```diff
@@ -0,0 +1,45 @@
+var merge = require('webpack-merge')
+var base = require('./index.js')
+
+module.exports = merge(base, {
+	app: {
```

Comment:
then `merge(base, dev-vm, {....`

And you'd only need to include items that need altering...app { publicPath }, server { memcached } etc...